doc/api/fs.md

```diff
@@ -4669,6 +4672,9 @@ Following successful read, the `Promise` is resolved with an object with a
 `bytesRead` property specifying the number of bytes read, and a `buffer`
 property that is a reference to the passed in `buffer` argument.
 
+If the file is not modified concurrently, the end-of-file is reached when the
+number of bytes read is smaller than the number of bytes requested.
```

Comment:
This wouldn’t be true for reads from non-blocking `fd`s, I think

Reply:
Should it be zero in that case? Any suggestions?

Reply:
I’m not 100 % sure but I think so, yeah

(From a quick test it looks like `fs.read()` on non-blocking `fd`s would give `EAGAIN` instead of a zero buffer when no data is available, so that would be consistent here)

Reply:
@addaleax PTAL.